doc/innovation_roadmap.rst

```diff
@@ -0,0 +1,218 @@
+     - Comparison with Open-TYNDP
+   * - The PEMMDB app will provide an API to allow efficient data transfer into the PLEXOS model 
+     - The Open TYNDP workflow already automates data integration though REST API calls to various sources, as well as automatically downloading, extracting, transforming and filtering data for all of its inputs.
+
+Quality Control
+===============
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1    
+
+   * - Innovation Roadmap Details
+     - Comparison with Open-TYNDP 
+   * - Perform sanity checks on each scenario, for example preventing simultaneous dispatch of electrolysers and H2/gas-fired plants, ensuring reasonable levels of curtailment and Energy Not Served, comparing generator margins, corss-sector assets and storage investment costs
+     - <Validation?>
+
+System Modelling Innovations
+============================
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1    
+
+   * - Innovation Roadmap Details
+     - Comparison with Open-TYNDP 
+   * - Hydrogen storage
+         - differentiate between short-term and seasonal storage needs
+         - better representation of operational constraints of storage facilities such as salt caverns and aquifers
+         - align national and modelling studies on storage capacities
+         - incorporate techno-economic constraints of hydrogen supply
+         - improve pipelines modelling to reflect transport flexibility
+     - PyPSA-Eur recently fixed bugs related to underground H2 cavern creation. Open-TYNDP includes regionalized H2 salt cavern potentials.
+   * - Integration of Hybrid heat pumps
+         - Ensure hybrid systems are correctly sized for applications, considering peak demand scenarios
+         - Ensure assumptions incorporate both economics and behavioural considerations
+     - PyPSA-Eur refined heat pump CAPEX allocations and COP approximations. It also introduced river- and sea-water sourced heat pumps.
+   * - Grid topology
+         - Incorporate more detailed representation of H2 network topology that approximates physical H2 flow to ensure key H2 corridors and infrastructure are well represented
+     - PyPSA-Eur supports administrative clustering (NUTS0 to NUTS3), allowing the network to be resolved at highly granular levels. Open-TYNDP uses the TYNDP H2 topology, including Z1 and Z2 setup.
+   * - Gas turbine usage and peaking unit utilisation 
+         - Explore dynamic operational needs of gas turbines given increasing reliance of variable renewable resources
+         - Update assumptions around CH4 to H2 retrofitting projects, which have struggled to compete in current markets
+     - Open TYNDP allows for endogenous retrofitting of CH4 pipeline capacity to H2.
+   * - EV Modelling
+         - Ensure that electricity flows follow charge/discharge cycles of EV batteries
+     - PyPSA-Eur refined Vehicle-to-Grid (V2G) dispatch capacity and temperature-dependent energy demand correction factors for EVs.
```

Comment:
- We do not see that charging/discharging behaviour at the same time (common problem in ESM) since we apply small marginal costs on the charging/discharging